src/utils/Logger.ts

```diff
@@ -182,7 +262,17 @@ class Logger {
       warn: console.info,
     }
 
-    const writeLog = (level: string, message: string) => {
+    const writeLog = async (level: string, message: string) => {
+      // If log folder not present create it
+      if (!(await RNFS.exists(this.getReactNativeLogsDir()))) {
+        await RNFS.mkdir(this.getReactNativeLogsDir())
+      }
+
+      // If daily log file is not present create it
+      if (!(await RNFS.exists(logFilePath))) {
```

Comment:
why check for existence?  i don't think we need to explicitly create the file (I think `append` will do that automatically for us).

Reply:
When I try to append without explicting creating the file I get the following error:
```
 Failed to write to .../Library/Caches/rn_logs/20220714.txt
 ```